cbcbeat/monodomainsolver.py

```diff
@@ -95,7 +95,15 @@ def __init__(self, mesh, time, M_i, I_s=None, v_=None,
 
         # Set-up function spaces
         k = self.parameters["polynomial_degree"]
-        V = FunctionSpace(self._mesh, "CG", k)
+        family = self.parameters["family"]
+
+        element = FiniteElement(
+            family=family,
+            cell=self._mesh.ufl_cell(),
+            degree=k,
+            quad_scheme="default",
```

Comment:
Do we actually need to send in the `"quad_scheme"`?

Reply:
I belive it will complain if family is "Quadrature" and we don't provide this argument (I don't exactly remember).